### ISSUE TITLE
Deliver sample metadata as a TSV

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#332]
### Context

As indicated in the discussion around sample metadata on https://github.com/AlexsLemonade/refinebio/issues/221 and by the TODO on https://github.com/AlexsLemonade/refinebio/pull/303, we'll want to deliver sample metadata in tab-delimited text format.

### Problem or idea

The smasher needs to be modified to deliver sample metadata in TSV form as well as JSON, which is currently implemented.

### Solution or next step

This should be addressed after #307 gets merged. More information can be found on the original ticket: #221 

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by jaclyn-taroni** [comment]
Closed by #347
